openedx/features/teams/views.py

```diff
@@ -123,7 +146,7 @@ def create_team(request, course_id, topic_id=None):
         'course': course,
         'user_has_privilege': not is_member_of_any_team,
         'countries': list(countries),
-        'languages': [[lang[0], _(lang[1])] for lang in settings.ALL_LANGUAGES],
+        'languages': [[lang[0], lang[1]] for lang in settings.ALL_LANGUAGES],
```

Comment:
Don't we have a way to avoid the error being raised here and still have the country names be translated?

Reply:
Ideally, `settings.ALL_LANGUAGES` should fetch string(s) as per system language.

For example, if the current language is Arabic then all languages (strings) must already be in Arabic inside `settings.ALL_LANGUAGES`